src/PrestaShopBundle/Resources/config/routing/admin/configure/shop_parameters/preferences.yml

```diff
@@ -8,7 +8,7 @@ admin_preferences:
 
 admin_preferences_save:
   path: preferences
-  methods: [ POST ]
```

Comment:
Ok for the PATCH method in the routing, but shouldn't the form and the controller be modified to use PATCH as well? This configuration allows PATCH requests but it doesn't mean it will use it just like this.
Is there a step missing here? Or is it somehow handed automatically somewhere?

Reply:
Here the patch method avoids the HTTP 405 `Method Not Allowed` error, when adding the patch method (as we just discussed, the patch method is already set in the multistore configuration form extension)